lib/index.js

```diff
@@ -72,6 +75,31 @@ internals.reformat = function () {
     }
 };
 
+internals.extendOutput = function (obj) {
+
+    // Underscore _.isObject
+    var isObject = function(obj){return (typeof obj === 'function' || typeof obj === 'object' && !!obj)};
```

Comment:
Seems kind of silly to declare two functions inside `extendOutput` that you only call once. Can you flatten this out?